### PR TITLE
Use Trusted task terminology

### PR DESCRIPTION
**Pull request by zregvart**
Reference: EC-390

### CODE REVIEW

**Comment by zregvart** [comment]
Make note of the second commit:

[Mass renaming of *acceptable to *trusted](https://github.com/enterprise-contract/ec-policies/pull/945/commits/0ae90b8ae498b6ccfef10a2f38ecc3324546c889)

Attention: this also changes rule short codes, which I think for these
rules should be okay.

File: policy/release/attestation_task_bundle.rego [hunk 1]
```diff
@@ -106,7 +106,7 @@ warn contains result if {
 #   For each Task in the SLSA Provenance attestation, check if the Tekton Bundle used is
 #   a trusted task.
 # custom:
-#   short_name: task_ref_bundles_acceptable
+#   short_name: task_ref_bundles_trusted
```
Comment (lcarva) [inline policy/release/attestation_task_bundle.rego:109]: I know of at least one user that excludes `attestation_task_bundle.task_ref_bundles_acceptable`. We could simply add the new name now to the policies in [here](https://gitlab.cee.redhat.com/releng/konflux-release-data/-/tree/main/config/OP/EnterpriseContractPolicy).

**Review by lcarva — COMMENTED** [review]
I think we should make this change, but let's see if we can first update the internal policies we are aware of.

**Review by lcarva — APPROVED** [review]

**Comment by zregvart** [comment]
Tracking rollout, there was an issue in the pipeline, just making sure the change has been deployed

**Comment by zregvart** [comment]
Validated in audit logs the new policy was applied.

**Pull request merged**